lib/prometheus/que_stats.rb

```diff
@@ -30,7 +30,7 @@ def execute
 
     class WorkerStats < Stats
       WORKER_STATS = <<~SQL
-        SELECT SUM(worker_count) AS workers_count, COUNT(*) AS nodes_count FROM que_lockers
+        SELECT COALESCE(SUM("worker_count"),0) AS workers_count, COUNT(*) AS nodes_count FROM que_lockers
```

Comment:
If no record or the `worker_count` column is `NULL` then `SUM` will return `nil`
The gem update validates the value is an number